superset-frontend/src/SqlLab/components/QueryHistory.tsx

```diff
@@ -31,21 +31,23 @@ interface QueryHistoryProps {
 
 const QueryHistory = ({ queries, actions, displayLimit }: QueryHistoryProps) =>
   queries.length > 0 ? (
-    <QueryTable
-      columns={[
-        'state',
-        'started',
-        'duration',
-        'progress',
-        'rows',
-        'sql',
-        'output',
-        'actions',
-      ]}
-      queries={queries}
-      actions={actions}
-      displayLimit={displayLimit}
-    />
+    <div className="scrollable">
```

Comment:
I'd suggest what Evan was demoing the other day and use emotion's css property here instead: https://codesandbox.io/s/emotion-playground-forked-7z07l?file=/src/index.js
That way you can reuse a scrollable style instead of a class.